utils/configcheck.py

```diff
@@ -52,6 +56,29 @@ def sd_configcheck(agentConfig):
         configs = load_check_directory(agentConfig, get_hostname(agentConfig))
         get_sd_configcheck(agentConfig, configs)
 
+def agent_container_inspect():
+    print("-- docker_inspect -- \n")
+    dockerutil = DockerUtil()
+    # Self inspection based on cgroups
+
+    proc_path = dockerutil._docker_root + '/proc/self/cgroup'
+    with open(proc_path, 'r') as fp:
+        lines = map(lambda x: x.split(':'), fp.read().splitlines())
+        subsystems = dict(zip(map(lambda x: x[1], lines), map(dockerutil._parse_subsystem, lines)))
```

Comment:
you don't need these 2 lines I think, you can just work on the full line. Just do
```python
for line in fp.read().splitlines():
    ...
```